### DISCUSSION TITLE
keep 数据下载

### DISCUSSION THREAD

**Discussion by YeasyHeart** [yihong0618/running_page#376] [Q&A] [answered]
keep 数据下载出现错误 
requests.exceptions.ProxyError: HTTPSConnectionPool(host='api.gotokeep.com', port=443): Max retries exceeded with url: /v1.1/users/login (Caused by ProxyError('Cannot connect to proxy.', OSError(0, 'Error')))

**Comment by yihong0618** [comment, marked as answer]
你本地有 proxy?

**Reply by YeasyHeart** [reply]
对，我开了vpn

**Reply by yihong0618** [reply]
关了试试。不要有 proxy
